app/controllers/spree/user_registrations_controller.rb

```diff
@@ -79,6 +79,13 @@ def after_sign_up_path_for(resource)
     after_sign_in_redirect(resource) if is_navigational_format?
   end
 
+  def after_inactive_sign_up_path_for(resource)
```

Comment:
are You using this method somewhere ? I suppose should be here: https://github.com/spree/spree_auth_devise/pull/478/files#diff-7c1521146da1200df72dc7ea1cd3aa36R100 ?

Reply:
@przemosk It's used in line 36 `respond_with resource, location: after_inactive_sign_up_path_for(resource)` when user is signed in verification is required.